### DISCUSSION TITLE
Discussion on "major issues / development areas"

### DISCUSSION THREAD

**Discussion by pcarruscag** [su2code/SU2#1486] [General]
If there is something you think fits this category, list it below and we will triage it appropriately for #1487

**Comment by bigfooted** [comment]
Wasn't there a wish list in one of the presentations of last year's meeting? I couldn't find it but I remember seeing it, maybe it was even longer ago.

## issues
- stability and convergence of multigrid (connected to #1362 )
- FFD box: mesh intersection prevention for 3D

## new developments
- axisymmetry for SA model
- monitoring points in the style of MARKER_ANALYZE and HISTORY_OUTPUT
- signal handling: make a call to the restart-file saving routines before terminating the program

**Reply by pcarruscag** [reply]
[This one?](https://docs.google.com/spreadsheets/d/1wgQDMiJ_mMqxiJ6FV8zZDu-CbxBjGXGeqrTAuJaTQkQ/edit?usp=sharing)
It's good for development areas but too broad for issues.

**Comment by tbellosta** [comment]
Don't know if this is outside the scope of the post, but I wouldn't mind seeing current development efforts listed here. At least major ones and those not yet in the develop branch. I'm aware of the SU2 Project branches page in the Projects tab, but I'm not sure if it is still maintained/updated.

**Reply by pcarruscag** [reply]
Thank you, I also think that would be important, but it is up to each individual / organization to report (if they want to) on the projects they are working on, the preferred ways to do that is with draft pull requests, github issues to track/document the design of a feature, and of course in person in our weekly developer meetings.
